ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
The point here is that the tags are _typed_, the way you set them is via:
```go
ext.SamplingPriority.Add(span, value)
```
and the compiler enforces the correct type of `value`

NB: we should change the syntax to use `Set` method instead of `Add`, it's a left-over from `AddTag` API